docs/guides/nwaku/configure-nwaku.md

```diff
@@ -0,0 +1,246 @@
+  --websocket-secure-key-path=privkey.pem \
+  --websocket-secure-cert-path=fullchain.pem \
+  --websocket-port=8002
+```
+
+:::tip
+You can use [Let's Encrypt](https://letsencrypt.org/) or [Certbot](https://certbot.eff.org/) to generate a valid certificate for your `nwaku` node:
+
+```bash
+sudo letsencrypt -d <your.domain.name>
+```
+:::
+
+## Configure REST API Server
+
+Nwaku provides a REST API to interact with the node and Waku Network. To enable the REST API, use the following configuration options:
+
+- `rest`: Enables the REST API server on the node (disabled by default).
+- `rest-address` (optional): Listening address of the REST API server. If you omit this option, it will default to `127.0.0.1`.
+- `rest-port` (optional): Listening port of the REST API server. If you omit this option, it will default to `8645`.
+- `rest-relay-cache-capacity`	(optional): Capacity of the Relay REST API message cache.	If you omit this option, it will default to `30`.
+- `rest-admin` (optional): Enables access to REST admin API (disabled by default).
+- `rest-private` (optional): Enables access to REST private API (disabled by default).
+
+```bash
+./build/wakunode2 \
+  --rest=true \
+  --rest-address=[REST SERVER LISTENING ADDRESS] \
+  --rest-port=[REST SERVER LISTENING PORT] \
+  --rest-relay-cache-capacity=[MESSAGE CACHE CAPACITY] \
+  --rest-admin=[true|false] \
+  --rest-private=[true|false]
+```
+
+For example, consider a `nwaku` node that enabled the REST API server on port `9000`:
+
+```bash
+./build/wakunode2 \
+  --rest=true \
+  --rest-port=9000 \
+  --rest-address=127.0.0.1
+```
+
+Consider a `nwaku` node that enabled the REST `admin` and `private` API with a message cache capacity of `100`:
+
+```bash
+./build/wakunode2 \
+  --rest=true \
+  --rest-admin=true \
+  --rest-private=true \
+  --rest-relay-cache-capacity=100
+```
+
+## Configure Filter Protocol
+
+To enable the [Filter protocol](/overview/concepts/protocols#filter) in `nwaku`, use the `filter` configuration option:
+
+```bash
+./build/wakunode2 --filter=true
+```
+
+You can configure `nwaku` as a `Filter client` using the `filternode` and `filter-timeout` options. This allows the node to request content filtering of messages from peers.
+
+```bash
+./build/wakunode2 \
+  --filternode=[FILTER PEER MULTIADDR] \
+  --filter-timeout=[FILTER PEER TIMEOUT]
+```
+
+For example, consider a `nwaku` node that requests content filtering of messages from peers with a timeout of `21600` seconds (6 hours):
+
+```bash
+./build/wakunode2 \
+  --filternode=/dns4/node-01.ac-cn-hongkong-c.wakuv2.prod.statusim.net/tcp/30303/p2p/16Uiu2HAm4v86W3bmT1BiH6oSPzcsSr24iDQpSN5Qa992BCjjwgrD \
+  --filter-timeout=21600
+```
+
+:::info
+If you omit the `filter-timeout` option, it will default to `14400` seconds (4 hours).
+:::
+
+## Configure Light Push Protocol
+
+To enable the [Light Push protocol](/overview/concepts/protocols#light-push) in `nwaku`, use the `lightpush` configuration option:
```

Comment:
I think best to specify this enables the "server side", ie, "to serve light clients"